src/main/java/htsjdk/samtools/liftover/LiftOver.java

```diff
@@ -125,10 +136,12 @@ public Interval liftOver(final Interval interval, final double liftOverMinMatch)
                 chainHit = chain;
                 targetIntersection = candidateIntersection;
             } else if (candidateIntersection != null) {
-                LOG.info("Interval " + interval.getName() + " failed to match chain " + chain.id +
-                " because intersection length " + candidateIntersection.intersectionLength + " < minMatchSize "
-                + minMatchSize +
-                " (" + (candidateIntersection.intersectionLength/(float)interval.length()) + " < " + liftOverMinMatch + ")");
+                if (logFailedIntervals){
```

Comment:
I think that a single (warn) message should still be emitted. something like:

"there are intervals that are failing to liftOver, but logFailedIntervals is false. To see what intervals have failed, re-run with logFailedIntervals=true."

Reply:
although this exposes the private member logFailedIntervals....so perhaps there's a better mechanism, but I would hate for there to be no indicatino at all that there are fialing intervals. The "correct" way to avoid information like this is to run at log_level=WARNING....